bin/versionPlugin.js

```diff
@@ -68,7 +68,7 @@ async function bumpStableTag(readmeTxt, version) {
 async function bumpVersionConstant(pluginFile, version) {
   return bumpVersion(
     pluginFile,
-    /^\s*\$this->define\(\s*'WPGRAPHQL_CONTENT_BLOCKS_VERSION', '([0-9.]+)/gm,
+    /^\s*define\(\s*'WPGRAPHQL_CONTENT_BLOCKS_VERSION', '([0-9.]+)/gm,
```

Comment:
while we're making changes to the versionPlugin.js. . .we might be able to satisfy the replacing of `@since @todo` references as well 🤔

Reply:
regular `@todo` like used in the `_doing_it_wrong( '...', '@todo' )` functions might be a bit broad. . .although we could regex replace that like we do the `WPGRAPHQL_CONTENT_BLOCKS_VERSION` above? 🤔

Reply:
That's outta my wheelhouse, I had to triple check this "simple" change 🙃

Reply:
haha! all good. I can follow-up on that separately. . .would just need to make a standard of how we add the placeholders. 

like is it always:  

```
_doing_it_wrong( 'some string', '@todo'  );
```

and

```
/**
 * Some docblock
 * @since @todo
 */
```

etc?

Reply:
I mean that's how _I_ do it, but really out of a general habit of checking _all_ `@todo`s before pushing a release.

To make it easier to regex regardless of if its `_doing_it_wrong()`, `_deprecated_*()`, `@since` or even a regular doc-block reference, maybe we intentionally want to use/reserve `@next` or `@next-version`? then we can just match the annotation instead of the wrapper

Reply:
ya, was thinking `@next-version` as well

Reply:
that way `@todo` can be left for other uses of `@todo` that IDEs already have tooling to pick up, etc 🤔

Reply:
So long as it doesnt make phpcs complain